v0/destinations/mautic/data/MauticIdentifyConfig.json

```diff
@@ -0,0 +1,104 @@
+[
+   
+    {
+      "sourceKeys": "email",
+      "destKey": "email",
+      "sourceFromGenericMap": true
+    },
+    {
+      "sourceKeys": "title",
```

Comment:
Title cannot be found at the root level of message. Please add traits.title or context.traits.title